extension/src/paymentHandler/payment-handler.js

```diff
@@ -26,6 +26,7 @@ async function handlePayment(paymentObject) {
     return { success: true, data: null }
 
   const paymentValidator = ValidatorBuilder.withPayment(paymentObject)
+    .validateRequiredFields()
```

Comment:
We will pass those to the metadata field of the Adyen payments payload, so I would rename those to 
`validateMetadataFields()` because everything is required for validation logic.

Reply:
c0fc985